packages/camera/camera_windows/windows/camera.cpp

```diff
@@ -50,24 +50,28 @@ CameraImpl::~CameraImpl() {
 bool CameraImpl::InitCamera(flutter::TextureRegistrar* texture_registrar,
                             flutter::BinaryMessenger* messenger,
                             bool record_audio,
-                            ResolutionPreset resolution_preset) {
+                            ResolutionPreset resolution_preset, int fps,
+                            int video_bitrate, int audio_bitrate) {
```

Comment:
The same set of arguments is being added to a bunch of methods, which is a strong indication that a simple data class to group related arguments would be useful here.

Reply:
That would also solve the significant usability/safety problem of having a bunch of positional arguments with the same types, which makes passing them incorrectly very easy. E.g., probably nobody would catch in review if you accidentally swapped `video_bitrate` and `audio_bitrate` at one of the layers, with the current signatures.

Reply:
wrapped using `struct RecordSettings`